src/pocketmine/Player.php

```diff
@@ -2713,8 +2723,27 @@ public function handlePlayerAction(PlayerActionPacket $packet) : bool{
 				}
 				return true;
 			case PlayerActionPacket::ACTION_START_GLIDE:
+				if($this->inventory->getChestplate()->getId() !== Item::ELYTRA){
+					$this->server->getLogger()->warning($this->getName() . " tried to start glide without Elytra");
```

Comment:
Add sending contents of chestplate to player (in case there were server<->client inventory inconsistencies)